includes/data-port/models/class-sensei-import-course-model.php

```diff
@@ -35,12 +35,33 @@ public function sync_post() {
 		$teacher = $this->get_default_author();
 
 		$teacher_username = $this->get_value( Sensei_Data_Port_Course_Schema::COLUMN_TEACHER_USERNAME );
+		$teacher_email    = $this->get_value( Sensei_Data_Port_Course_Schema::COLUMN_TEACHER_EMAIL );
 
 		if ( ! empty( $teacher_username ) ) {
-			$teacher = Sensei_Data_Port_Utilities::create_user( $teacher_username, $this->get_value( Sensei_Data_Port_Course_Schema::COLUMN_TEACHER_EMAIL ), 'teacher' );
+			$teacher_user = Sensei_Data_Port_Utilities::create_user( $teacher_username, $teacher_email, 'teacher' );
 
-			if ( is_wp_error( $teacher ) ) {
-				return $teacher;
+			if ( is_wp_error( $teacher_user ) ) {
+				return $teacher_user;
```

Comment:
Although it's not from this PR, I wonder if we should only add a warning here, instead of don't create the course. Checking @donnapep's spreadsheet, I didn't find this case to don't create the course.

Reply:
I imagine that we can go either way. `wp_create_user` will return `WP_Error` in various cases like for example the email belonging to another user or the username being too long. Any ideas @donnapep?

Reply:
Yeah, could go either way, so might as well do what's easiest and leave it as is.